viadot/tasks/prefect_conformed.py

```diff
@@ -0,0 +1,147 @@
+import prefect
+from datetime import datetime, timedelta
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+from prefect.utilities import logging
+
+logger = logging.get_logger()
+
+class ReRunFailedFlow(Task):
+    def __init__(
+        self,
+        flow_name: str = None,
+        *args,
+        **kwargs,
+    ):
+
+        self.flow_name = flow_name
+
+        super().__init__(
+            name="prefect_extract_details",
+            *args,
+            **kwargs,
+        )
+
+    def __call__(self):
+        """Extract time from Prefect Flow run"""
+        super().__call__(self)
+    
+    def check_if_scheduled_run(self, created_by_user_id: str = None) -> bool:
+        """
+        Check if run was scheduled or started by user.
+
+        Args:
+            created_by_user_id (str, optional): User_id in prefect. Defaults to None.
+
+        Returns:
+            bool: True if flow run was started automatically. False if Flow was started by user.
+        """
+
+        if created_by_user_id == "09720c91-a99c-4f72-b7b5-3c061c83408b" :
+            return True
+        else:
+            return False
```

Comment:
Same situation as `get_formatted_date` -> maybe is possible to add option to checking also by `user_id`. In `prefect.py` is  `time_run` and `time_schedule` - this is the first option. 
Your option can work like:  if time_run and time_schedule is None then check by `user_id` - only suggestion